Runtime/Scripts/RGServiceManager.cs

```diff
@@ -220,6 +226,7 @@ public async Task GetRunningInstancesForBot(long botId, Action<RGBotInstance[]>
                 },
                 onFailure: async (f) =>
                 {
+                    RGDebug.LogWarning($"Failed to get running instance for bot: {f}");
```

Comment:
This seems valid, but very spammy.  Instead, we should handle this in `RGOverlayMenu.UpdateBots` and consolidate the errors into one log statement after all the calls finish.

Reply:
I like that idea - I consolidated them in the calling method